katello-pull-transport-migrate

```diff
@@ -1,14 +1,9 @@
 #!/bin/sh
 
-# set SBINDIR to /sbin if it is not set
-if [ -z "$SBINDIR" ]; then
-    SBINDIR=/usr/sbin
-fi
-
 # check if system is registered
 subscription-manager identity > /dev/null
 if [ $? -ne 0 ]; then
-    echo "Please register with subscription-manager first, then re-run `$SBINDIR/katello-pull-transport-migrate`."
+    echo "Please register with subscription-manager first, then re-run `$0`."
```

Comment:
I also made this change, and more serious bug fixes in #2, such as the infinite recursion you have here on this exact line.

Reply:
Okay then. I'm reverting the changes to the script. Let's keep those in #2.